docs/HowToSupportTizenRtProtectedBuild.md

```diff
@@ -0,0 +1,250 @@
+# Steps for Supporting Protected Build in a new chipset
```

Comment:
Could n't understand why protected-build is tied with MPU?
Any processor which supports different privilege levels can support protected build? 
What is the definition for protected build? If its just user & kernel mode, then its NOT mandatory to have MPU.

Reply:
Further, since protected build is NOT a common name, if MPU is mandatory. Then please add the definition of protected-build very first.

Reply:
The explanation regarding what is protected build is given in the ProtectedBuildGuide.md. 
It is true that it is not mandatory to have MPU for protected build. But, whenever the microcontroller supports an MPU, it is preferable to use it. We will update the ProtectedBuildGuide.md accordingly. We have included the sections for MPU in this document because, most cortex M projects that we might work in future will most probably support an MPU and it would be better to make use of it.